kubespawner/spawner.py

```diff
@@ -1008,6 +1003,9 @@ def get_pvc_manifest(self):
         Make a pvc manifest that will spawn current user's pvc.
         """
         labels = self._build_common_labels(self._expand_all(self.user_storage_extra_labels))
+        labels.update({
+            'component': 'singleuser-storage'
+        })
```

Comment:
The call to `_build_common_labels` here is the only other place `_build_common_labels` is used. So the PVCs will get all the recommended labels as well.